### DISCUSSION TITLE
Integration of alerts from third party tool to the security onion interface

### DISCUSSION THREAD

**Discussion by MAALAOUI-M** [Security-Onion-Solutions/securityonion#11348] [2.4]
### Version

2.4.10

### Installation Method

Security Onion ISO image

### Description

other (please provide detail below)

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

4

### RAM

64

### Storage for /

1To

### Storage for /nsm

1 To

### Network Traffic Collection

span port

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Hello !
I hope you're all doing well. I have a question regarding the integration of alerts from a third-party tool into Security Onion. I've recently started using a third-party security tool that generates alerts related to vulnerabilities, and I'd like to centralize the monitoring of these alerts within my Security Onion environment.

The third-party tool provides alerts in a specific format, and I'm looking for guidance on how to feed these alerts into Security Onion so that I can leverage its powerful monitoring and analysis capabilities.

If anyone in the community has experience with similar integrations or can provide insights into how I can achieve this, I would greatly appreciate your guidance and advice.

Thanks !

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by tomjones1977** [comment]
There's very limited information in your post, but it's possible to ingest logs using elasticsearch pipelines.  

https://docs.securityonion.net/en/2.3/elasticsearch.html#custom-templates

Depending on the format of the data being sent from the third party service you will probably need to write/use a custom ingest pipeline to get it into the right format for elasticsearch.

**Comment by jaypfe** [comment]
I would first check to see if there is an Elastic integration available for your tool.  If not, see if it allows log forwarding via syslog. In either case that should be your first step. 

If there is an integration available, be advised that it takes a little messing with to get it working for the time being. see #10942
